roxctl/analyze/netpol/netpol_test.go

```diff
@@ -104,6 +112,51 @@ func (d *analyzeNetpolTestSuite) TestAnalyzeNetpol() {
 			expectedValidateError: nil,
 			expectedAnalysisError: nil,
 		},
+		{
+			name:                  "not supported output format",
+			inputFolderPath:       "testdata/minimal",
+			outputFormat:          "docx",
+			errStringContainment:  true,
+			expectedAnalysisError: errors.New("docx output format is not supported."),
+		},
```

Comment:
bool flag  `errStringContainment` was added here to help comparing the `expectedAnalysisError`  (which is `errorstring` returned by `ConnectionsListToString(conns)` in this case) with the actual error ( see line 209).

I didn't succeed to compare it using `ErrorIs ` (didn't figure how to compare this error type with in chain error)